commit 9702424ea23471e3bdbe6a150e28a1ecdf837d2c
Author: PJBrs <pjbrs@floorenpj.xs4all.nl>
Date:   Fri Feb 21 08:56:04 2020 +0100

    ARM: dts: sunxi: h2-plus-bananapi-m2-zero: Add HDMI out
    
    Add HDMI out, including the display engine, to the BananaPi
    M2 Zero.

diff --git a/arch/arm/boot/dts/sun8i-h2-plus-bananapi-m2-zero.dts b/arch/arm/boot/dts/sun8i-h2-plus-bananapi-m2-zero.dts
index 38e8da594e44..9051a1097258 100644
--- a/arch/arm/boot/dts/sun8i-h2-plus-bananapi-m2-zero.dts
+++ b/arch/arm/boot/dts/sun8i-h2-plus-bananapi-m2-zero.dts
@@ -26,6 +26,17 @@
 		stdout-path = "serial0:115200n8";
 	};
 
+	connector {
+		compatible = "hdmi-connector";
+		type = "a";
+
+		port {
+			hdmi_con_in: endpoint {
+				remote-endpoint = <&hdmi_out_con>;
+			};
+		};
+	};
+
 	leds {
 		compatible = "gpio-leds";
 
@@ -140,6 +151,20 @@
 	};
 };
 
+&de {
+	status = "okay";
+};
+
+&hdmi {
+	status = "okay";
+};
+
+&hdmi_out {
+	hdmi_out_con: endpoint {
+		remote-endpoint = <&hdmi_con_in>;
+	};
+};
+
 &ohci0 {
 	status = "okay";
 };
